pkg/ctl/create/cluster.go

```diff
@@ -46,11 +45,13 @@ func createClusterCmd(g *cmdutils.Grouping) *cobra.Command {
 	cfg := api.NewClusterConfig()
 	ng := cfg.NewNodeGroup()
 
+	ngFilter := NewNodeGroupFilter()
```

Comment:
This was mostly to mirror `createNodeGroupCmd`/`doCreateNodeGroup`, but also eventually we'd like to add `--only` to all of the commands (see [upgrades proposal doc](https://github.com/weaveworks/eksctl/blob/master/docs/proposal-005-upgrades.md#cli-design), for example). We might rename the filter, but it would need to be set based on flags, so it seems to makes sense to keep it here.